sourcegraph/cody
# Add support link for Cody Pro & Enterprise

Adds Support to the sidebar and settings quickpick:

![CleanShot 2024-03-07 at 12 40 16@2x](https://github.com/sourcegraph/cody/assets/153/a3ae9ece-fe4d-4043-a347-3408e91796e8)

![CleanShot 2024-03-07 at 12 40 23@2x](https://github.com/sourcegraph/cody/assets/153/687db7c6-1f00-4e4d-a3d5-5e812affa629)

- [x] Fix Support to only show for Pro & Ent in quickpick

## Test plan

- Open sidebar on pro & ent plans. Click Support. Verify works.
- Open settings quickpick on pro & ent plans. Click Support. Verify works.
- Open sidebar on free plan. Verify no support link.
- Open settings quickpick on free plan. Verify no support link.
